src/pixel.rs

```diff
@@ -1,6 +1,7 @@
 //! Encloses pixel-related traits and pixel type implementations.
 
 use crate::Error::DecodingError;
+use crate::arch;
```

Comment:
Should be merged with the import below, i.e. `use crate::{arch, encodings::ColorType, ...}`
Also the `use crate::Error::DecodingError` on line 3 could be moved there too

Reply:
ok 3332